massa-ledger/src/test_exports/config.rs

```diff
@@ -2,14 +2,43 @@
 
 /// This file defines testing tools related to the config
 use crate::LedgerConfig;
+use massa_models::{
+    constants::default_testing::{FINAL_HISTORY_LENGTH, THREAD_COUNT},
+    Address, Amount,
+};
+use std::collections::BTreeMap;
+use std::io::prelude::*;
+use tempfile::NamedTempFile;
 
 /// Default value of LedgerConfig used for tests
 impl Default for LedgerConfig {
     fn default() -> Self {
         LedgerConfig {
-            initial_sce_ledger_path: "".into(), // unused by the mock
-            final_history_length: 10,
-            thread_count: 2,
+            // unused by the mock (you can use `LedgerConfig::sample()` to get
+            // a NamedTempFile in addition)
+            initial_sce_ledger_path: "".into(),
+            final_history_length: FINAL_HISTORY_LENGTH,
+            thread_count: THREAD_COUNT,
         }
     }
 }
+
+impl LedgerConfig {
```

Comment:
Great! This is really the kind of utility methods we want everywhere in the codebase :slightly_smiling_face: